ansible/roles/ci/doc/setup/semaphore.md

```diff
@@ -0,0 +1,29 @@
+# Semaphore Server
+
+## How to setup
+
+The following repository can be used as an example of [Semaphore](https://semaphoreci.com/) configuration [build servers check](https://github.com/shadow-robot/build-servers-check)
```

Comment:
"The following repository can be used as an example of a [Semaphore](https://semaphoreci.com/) configuration: [build servers check](https://github.com/shadow-robot/build-servers-check)."